Formula/tendermint.rb

```diff
@@ -15,8 +15,8 @@ def install
 
   test do
     shell_output("#{bin}/tendermint init --home /tmp")
```

Comment:
This isn't always usable. I think this would be better.
```suggestion
    mkdir (testpath/"staging")
    shell_output("#{bin}/tendermint init --home #{testpath}/staging")
```

Reply:
+1

Reply:
Cool. This will also be cleaned up afterwards right?

Reply:
Yes it will